src/components/macroview/centerPanel/SequencingArea.tsx

```diff
@@ -38,6 +40,8 @@ export default function SequencingArea({ onOpenSettingsModal }: Props) {
   } = useMacroContext()
   const { config } = useSettingsContext()
   const { isOpen, onOpen, onClose } = useDisclosure()
+  const fontSize = '0.844rem' //REM size for the font (13.5 px)
```

Comment:
this is pretty sus ngl

Reply:
I think I consulted this change with Khang and explained rem is more consistent. Maybe there's a predefined size thats better for this?

Reply:
rem is better, but I would much rather use a named value than some arbitrary value

Reply:
I tried using 'xs', that looks pretty good and it fixed a clipping issue with the icon.

[ec7f9a4](https://github.com/WootingKb/wooting-macros/pull/194/commits/ec7f9a4bbb416492597b100fbe57aec623287ac7)

Reply:
In that case, it looks like you don't need it as a separate variable anymore and it should just be inline in the JSX

Reply:
Removed: [75a90f3](https://github.com/WootingKb/wooting-macros/pull/194/commits/75a90f3b61a782b6985bac6e86b738b0c207330d)